js/web/lib/wasm/jsep/webgpu/ops/gather-elements.ts

```diff
@@ -32,65 +32,68 @@ const createGatherElementsProgramInfo =
       const inputShape = inputs[0].dims;
       const inputOutputDataType = inputs[0].dataType;
       const inputRank = inputShape.length;
-      const inputStrides = ShapeUtil.computeStrides(inputShape);
-      const inputSize = ShapeUtil.size(inputShape);
 
       const indicesShape = inputs[1].dims;
       const indicesDataType = inputs[1].dataType;
-      const indicesSize = ShapeUtil.size(indicesShape);
-
       const axis = ShapeUtil.normalizeAxis(attributes.axis, inputRank);
       const axisDimLimit = inputShape[axis];
 
       const outputShape = indicesShape.slice(0);
       const outputSize = ShapeUtil.size(outputShape);
+      const enableInputShapesUniforms = enableShapesUniforms(inputShape.length);
```

Comment:
`enableShapesUniforms` is not needed anymore since ND uniforms are supported now.